src/main/java/at/hannibal2/skyhanni/config/features/GUIConfig.java

```diff
@@ -56,6 +56,26 @@ public static class TextBoxConfig {
     @Expose
     public Position realTimePosition = new Position(10, 10, false, true);
 
+    @Expose
+    @ConfigOption(name = "In-game Date", desc = "Show the in-game date of Skyblock (like in Apec, §ebut with mild delays§7).\n(Though this one includes the Skyblock year!)")
```

Comment:
Both uses of "Skyblock" should be "SkyBlock" for consistency sake.

Reply:
a simple control+F says yes

however, observe the lowercase `B` the admins used on akinsoft: https://sky.shiiyu.moe/stats/Akinsoft/Not%20Allowed%20To%20Quit%20Skyblock%20Ever%20Again

Reply:
but fine, i'll revise it

Reply:
I prefer the wording of the game mode in the Hypixel game selector for everyone  over an troll/Easter egg/bullying/one time joke for one player.